north/src/linux/mount.rs

```diff
@@ -146,18 +146,9 @@ pub async fn mount(
 }
 
 pub async fn unmount(target: &Path) -> Result<()> {
-    let c_target = CString::new(target.as_os_str().as_bytes())?;
-
-    task::spawn_blocking(move || {
-        let result = unsafe { libc::umount(c_target.as_ptr()) };
-
-        match result {
-            0 => Ok(()),
-            _err => Err(anyhow!(
-                "Failed to unmount: {}",
-                std::io::Error::last_os_error()
-            )),
-        }
-    })
-    .await
+    let path = target.to_owned();
+    task::spawn_blocking(move || nix::mount::umount(path.as_os_str()))
```

Comment:
`mount` uses `libc`. `umount` uses `nix`. This should be the same for both not mix the deps in one mod.